whisper.h

```diff
@@ -383,8 +383,10 @@ extern "C" {
 
     // Temporary helpers needed for exposing ggml interface
 
-    WHISPER_API int whisper_bench_memcpy(int n_threads);
-    WHISPER_API int whisper_bench_ggml_mul_mat(int n_threads);
+    WHISPER_API int whisper_print_bench_memcpy(int n_threads);
+    WHISPER_API const char * whisper_bench_memcpy(int n_threads);
+    WHISPER_API int whisper_print_bench_ggml_mul_mat(int n_threads);
+    WHISPER_API const char * whisper_bench_ggml_mul_mat(int n_threads);
```

Comment:
Please rename as follows:

```suggestion
    WHISPER_API int whisper_bench_memcpy(int n_threads);
    WHISPER_API const char * whisper_bench_memcpy_str(int n_threads);
    WHISPER_API int whisper_bench_ggml_mul_mat(int n_threads);
    WHISPER_API const char * whisper_bench_ggml_mul_mat_str(int n_threads);
```

Even though these are only temporary, I prefer to not change the existing API.